api/ccapi/include/tensor_dim.h

```diff
@@ -95,7 +113,7 @@ class TensorDim {
    * @brief     Creator of TensorDim with Format & DataType
    *
    * @param fm format NCHW | HNWC
-   * @param fm DataType QINT4 | QINT8 | FP16 | FP32
+   * @param d_type DataType QINT4 | QINT8 | FP16 | FP32
```

Comment:
The data type is defined as d_type.
However, the tensor type was defined as t_type_.
Is there a reason for this naming?

Reply:
I think it is to differentiate with member variable TensorType is named `t_type`. I know this is opposed to the Google C++ Style guide (Data members of classes with trailing underscore). Let's make a separate PR to unify all the symbol names in NNTrainer following the same naming convention.